components/autocomplete/index.js

```diff
@@ -249,9 +249,17 @@ export class Autocomplete extends Component {
 	search( event ) {
 		const { open: wasOpen } = this.state;
 		const { completers } = this.props;
-		// ensure that the cursor location is unambiguous
 		const container = event.target;
+		// check that the event came from a contentEditable
+		// Note that the hasAttribute/getAttribute is because JsDOM does not support container.contentEditable
+		if ( ! container.hasAttribute( 'contentEditable' ) || container.getAttribute( 'contentEditable' ) === 'false' ) {
+			return;
+		}
+		// ensure that the cursor location is unambiguous
 		const cursor = this.getCursor( container );
+		if ( ! cursor ) {
+			return;
```

Comment:
When is this reached, and how does it relate to the intent of the pull request?

Reply:
It is reached when the selection is not collapsed which apparently happens with some tests that Morgan was working on.